DailyHealth/CustomerDisplay.java

```diff
@@ -0,0 +1,1085 @@
+					targetWeightLabel.setText(newTargetWeightStr);
+				}else if(Current(newTargetWeightStr) == false){
+					JOptionPane.showMessageDialog(null, "목표 몸무게를 변경하지 못했습니다", "목표 몸무게 변경", JOptionPane.ERROR_MESSAGE);
+				}
+			}
+		});
+		
+		// bmi 계산 버튼
+		JButton bmiBtn = new JButton("bmi 지수 계산");
+		bmiBtn.setBounds(480, 385, 250, 50);
+		customerPanel.add(bmiBtn);
+		bmiBtn.setBackground(Color.green);
+		bmiBtn.setFont(new Font("맑은 고딕", Font.PLAIN, 22));
+		bmiBtn.addActionListener(new ActionListener() {
+			@Override
+			public void actionPerformed(ActionEvent e) {
+				BmiFrame bmi = new BmiFrame();
+			}
+		});
+//		arrow.setIcon(new ImageIcon("./image/weight_arrow.png"));
+//		arrow.setBorderPainted(false);
+//		arrow.setContentAreaFilled(false);
+//		arrow.setFocusPainted(false);
+		
+		// bmr 계산 버튼
+		JButton bmrBtn = new JButton("기초대사량(bmr) 계산");
+		bmrBtn.setBounds(480, 455, 250, 50);
+		customerPanel.add(bmrBtn);
+		bmrBtn.setBackground(Color.green);
+		bmrBtn.setFont(new Font("맑은 고딕", Font.PLAIN, 22));
+		bmrBtn.addActionListener(new ActionListener() {
+			@Override
+			public void actionPerformed(ActionEvent e) {
+				BmrFrame bmr = new BmrFrame();
+			}
+		});
+//		arrow.setIcon(new ImageIcon("./image/weight_arrow.png"));
+//		arrow.setBorderPainted(false);
+//		arrow.setContentAreaFilled(false);
+//		arrow.setFocusPainted(false);
+		
+		// 로그아웃 버튼
+		JButton logoutBtn = new JButton("로그아웃");
+		logoutBtn.setBounds(870, 510, 105, 35);
+		customerPanel.add(logoutBtn);
+		logoutBtn.setBackground(Color.green);
+		logoutBtn.setFont(new Font("맑은 고딕", Font.PLAIN, 17));
+		logoutBtn.addActionListener(new ActionListener() {
+			@Override
+			public void actionPerformed(ActionEvent e) {
+				frame.setVisible(false);
+				new LogoChoice();
+			}
+		});
+	}
+	public static void main(String[] args) {
+		new CustomerDisplay();
+	}
+	
+	// 출석체크 메서드
+	public boolean attend(String id) {
+		String query;
+		PreparedStatement pstmt = null;
+		Connection conn = null;
+		ResultSet rset = null;
+		
+        try {
+            Class.forName("com.mysql.jdbc.Driver");
+            conn = DriverManager.getConnection("jdbc:mysql://localhost:3306/daily_health?serverTimezone=UTC", "root", "password");
+            
+            PreparedStatement pst=conn.prepareStatement("update jiyoon set remain_days=? where id=?");
+
+            pst.setDouble(1, Integer.parseInt(remainDaysLabel.getText()) - 1);
+            pst.setString(2, id);
+
+            int numRows = pst.executeUpdate();
+
+            System.out.format("출석체크 : %d개의 행이 바꼈습니다.", numRows);
+
+            pst.close();
+            conn.close();
+			
+            return true;
+        } catch (ClassNotFoundException e) {
+            e.printStackTrace();
+            return false;
+        } catch (SQLException e) {
+            e.printStackTrace();
+            return false;
+        }
+	}
+	
+	// 현재 몸무게 변경 메서드
+	public boolean Current(String cw) {		

```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.